global.json

```diff
@@ -3,13 +3,18 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "7.0.304",
+    "dotnet": "8.0.100-preview.3.23178.7",
```

Comment:
Is there a reason why you don't update to the latest officially released preview (P6)?

Reply:
Doing this incrementally is fine as well, was just wondering.

Reply:
Pretty sure it's "Rainer started this a long time ago and we haven't updated" but I also lean to doing it incrementally (given that it's almost working now).